src/content/developers/docs/smart-contracts/formal-verification.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Formal verification of smart contracts
+description: An overview of formal verification techniques for Ethereum smart contracts 
+lang: en
+sidebar: true
+---
+
+
+## A brief introduction {#introduction-to-formal-verification}
+
+[Smart contracts]([/smart-contracts/](https://ethereum.org/en/developers/docs/smart-contracts/)) are making it possible to create decentralized, trustless, and robust applications that introduce new use-cases and unlock value for users. Because smart contracts handle large amounts of value, security is a critical consideration for developers.
```

Comment:
```suggestion
[Smart contracts]([/smart-contracts/](/developers/docs/smart-contracts/)) are making it possible to create decentralized, trustless, and robust applications that introduce new use-cases and unlock value for users. Because smart contracts handle large amounts of value, security is a critical consideration for developers.
```
Note on relative links: https://ethereum.org/en/contributing/style-guide/#internal-links

Reply:
Ouch. Must have missed it while editing. Looks like I'll have to go through the guide again.